src/renderer/components/ft-profile-bubble/ft-profile-bubble.js

```diff
@@ -22,7 +22,12 @@ export default Vue.extend({
   },
   computed: {
     profileInitial: function () {
-      return this.profileName.slice(0, 1).toUpperCase()
+      if (this.profileName !== undefined) {
+        if (this.profileName.length > 0) {
+          return Array.from(this.profileName)[0].toUpperCase()
+        }
+      }
+      return ''
```

Comment:
Simplify this to:
```js
return this?.profileName?.length > 0
      ? Array.from(this.profileName)[0].toUpperCase()
      : ''
```
Do it similarly on the other cases.

Reply:
Didn't realize optional chaining was a thing pretty cool.